test/fixtures/responses/get-ledger-with-partial-payment.json

```diff
@@ -94,12 +89,7 @@
           }
         },
         "destination": {
-          "address": "rNNuQMuExCiEjeZ4h9JJnj5PSWypdMXDj4",
-          "amount": {
-            "currency": "USD",
-            "value": "10",
-            "counterparty": "rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B"
-          }
+          "address": "rNNuQMuExCiEjeZ4h9JJnj5PSWypdMXDj4"
         },
         "allowPartialPayment": true
       },
```

Comment:
I'm wondering why this transaction doesn't have a `deliveredAmount`

Reply:
Fixed